manifests/4.6/logforwardings.crd.yaml

```diff
@@ -72,6 +72,9 @@ spec:
                       - logs.app
                       - logs.infra
                       - logs.audit
+                  labels:
+                    type: map
+                    description: Pipeline labels in the same format as medata.labels
```

Comment:
Does declaring labels having the same format as `metadata.labels` imply the same constraints? If yes, we should add some validation on this field (See k8s metadata labels contraints here: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)

If not, just remove the reference `metadata.labels` and re-phase to `key value pairs`

Reply:
Fixed